### PR TITLE
SM/GDRCOPY/UGNI: configure.m4 cleanup

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8481/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@bureddy @MattBBaker @brminich can you pls take a look?

**Comment by MattBBaker** [comment]
👍

File: src/uct/sm/knem/configure.m4 [hunk 1]
```diff
@@ -1,38 +1,37 @@
 #
-# Check for KNEM support
+# Copyright (C) Mellanox Technologies Ltd. 2001-2018.  ALL RIGHTS RESERVED.
```
Comment (bureddy) [inline src/uct/sm/knem/configure.m4]: 2019?

**Pull request merged**